### ISSUE TITLE
Tagged Versions? 

### ISSUE THREAD

**Issue by jrdemasi** [gpertea/gffcompare#30]
It would be great if you could identify tagged releases of gffcompare and gclib that will work to build this, as I don't want to use an untagged version on our compute cluster.  Also, for what it's worth, adding a flag to identify where gclib is would definitely beat hard coding "../gclib".

**Comment by gpertea** [comment]
Agreed, I'm still trying to figure out the best way to easily share the gclib code across multiple projects while coordinating release, tagging etc with git. gclib is quite eclectic and I don't even need all the source files in there for every project.. Any suggestions/pointers about doing this neatly and optimally are welcome.
Not sure what you meant by using a flag for the gclib location. Since I'm not planning to mess with the autoconf stuff anytime soon, I suppose just with make I could do something like:
    
     make GCLIB=/path/to/gclib release

Would that suffice? (just applied this little change to the master branch/latest tag)

**Comment by jrdemasi** [comment]
It's certainly a step in the right direction, and would fit my use case.  It gets hard with utility libraries like that!  As long as I can install in `/opt/gclib/<version>` and `/opt/gffcompare/<version>` my use case would be satisfied. 

At my institution in particular, we won't usually install software that is just "latest" unless it's tagged.  So if you had two tagged releases that work together, I'd be able to install it centrally for my users.

**Comment by jrdemasi** [comment]
Oh! Also maybe consider a submodule for gclib.
